webui/react/src/e2e/tests/projects.spec.ts

```diff
@@ -94,46 +95,117 @@ test.describe('Project UI CRUD', () => {
     });
   });
 
-  test('Move a Project', async ({
-    authedPage,
-    newWorkspace,
-    backgroundApiWorkspace,
-    backgroundApiProject,
-  }) => {
-    const workspaceDetails = new WorkspaceDetails(authedPage);
+  test.describe('Move Project', () => {
+    let newProjectCard: V1PostProjectResponse;
+    let newProjectRow: V1PostProjectResponse;
+    let destinationWorkspace: V1Workspace;
+    let originWorkspace: V1Workspace;
+
+    test.beforeAll(async ({ backgroundApiProject, backgroundApiWorkspace }) => {
+      originWorkspace = (await backgroundApiWorkspace.createWorkspace(backgroundApiWorkspace.new()))
+        .workspace;
+      destinationWorkspace = (
+        await backgroundApiWorkspace.createWorkspace(backgroundApiWorkspace.new())
+      ).workspace;
+
+      newProjectCard = await backgroundApiProject.createProject(
+        originWorkspace.id,
+        backgroundApiProject.new(),
+      );
+      newProjectRow = await backgroundApiProject.createProject(
+        originWorkspace.id,
+        backgroundApiProject.new(),
+      );
+
+      projectIds.push(newProjectCard.project.id);
+      projectIds.push(newProjectRow.project.id);
+      workspaceIds.push(destinationWorkspace.id);
+      workspaceIds.push(originWorkspace.id);
+    });
+    test.beforeEach(async ({ authedPage }) => {
+      const workspaceDetails = new WorkspaceDetails(authedPage);
+      await workspaceDetails.gotoWorkspace(originWorkspace.id);
+    });
+    test.afterAll(async ({ backgroundApiProject, backgroundApiWorkspace }) => {
+      for (const workspaceId of workspaceIds) {
+        await backgroundApiWorkspace.deleteWorkspace(workspaceId);
+      }
```

Comment:
if there's no `originWorkspace`, could remove `workspaceIds` array and just delete `destinationWorkspace` in this `afterAll`. Otherwise, can define the `workspaceIds` array in the `Move Project` block's scope.